fluid/image_classification/README.md

```diff
@@ -6,3 +6,68 @@ The minimum PaddlePaddle version needed for the code sample in this directory is
 
 This model built with paddle fluid is still under active development and is not
 the final version. We welcome feedbacks.
+
+## Introduction
+
+The current code support the training of [SE-ResNeXt](https://arxiv.org/abs/1709.01507) (50/152 layers).
+
+## Data Preparation
+
+1. Download ImageNet-2012 dataset
+```
+# get training set
+wget XXX
+# get validation set
+wget YYY
+# prepare directory
+mkdir ILSVRC2012/
+tar zxf XXX
+tar zxf YYY
+```
+
+2. Generate training and validation file
+
+```train_list.txt``` and ```test_list.txt``` are generated for data provider in ```reader.py```. Lines in the two files are like:
+```
+ILSVRC2012_val_00000003.JPEG 230
+ILSVRC2012_val_00000004.JPEG 809
+ILSVRC2012_val_00000005.JPEG 516
+ILSVRC2012_val_00000006.JPEG 57
+...
+```
+```
+n01440764/n01440764_13575.JPEG 0
+n01440764/n01440764_13581.JPEG 0
+n01440764/n01440764_13602.JPEG 0
+n01440764/n01440764_13625.JPEG 0
+...
+```
```

Comment:
Can these file lists download in somewhere ? like https://github.com/BVLC/caffe/blob/master/data/ilsvrc12/get_ilsvrc_aux.sh